privacy/enforcement_test.go

```diff
@@ -52,7 +52,6 @@ func TestAny(t *testing.T) {
 func TestApply(t *testing.T) {
 	testCases := []struct {
```

Comment:
I realize this might be out of the scope of this PR but maybe we can expand the tests to a couple more scenarios. For instance, when all 3 `CCPA`, `COPPA`, and `GDPR` are false, or switching around some other relevant variables.
|   | CCPA | COPPA | GDPR | expectedDeviceMacAndIFA | expectedDeviceIPv6        | expectedDeviceGeo               | expectedUser                | expectedUserGeo                 | description                 |
|:-:|:----:|:-----:|:----:|:-----------------------:|:-------------------------:|:-------------------------------:|:---------------------------:|:-------------------------------:|:---------------------------:|
|  1| true | true  | true | true                    | ScrubStrategyIPV6Lowest32 | ScrubStrategyGeoFull            | ScrubStrategyUserFull       | ScrubStrategyGeoFull            | "All Enforced - Most Strict"|
|  2|false | true  |false | true                    | ScrubStrategyIPV6Lowest32 | ScrubStrategyGeoFull            | ScrubStrategyUserFull       | ScrubStrategyGeoFull            | "COPPA"                     |
|  3| false| false | true | false                   | ScrubStrategyIPV6Lowest16 | ScrubStrategyGeoReducedPrecision| ScrubStrategyUserBuyerIDOnly| ScrubStrategyGeoReducedPrecision| "GDPR"                      |
|  4| true | false |false | false                   | ScrubStrategyIPV6Lowest16 | ScrubStrategyGeoReducedPrecision| ScrubStrategyUserBuyerIDOnly| ScrubStrategyGeoReducedPrecision| "CCPA"                      |

Reply:
Good point. I'll address this is in my next PR. I'm unifying a lot of the special cases which exist today.